modules/logging-loki-retention.adoc

```diff
@@ -1,19 +1,21 @@
 // Module included in the following assemblies:
-//
-// logging/cluster-logging-loki.adoc
```

Comment:
Just an FYI that it seems like the assembly in this line still has this module in it, if that's intentional I'd recommend putting this comment back

Reply:
Yes the module is still included in `cluster-logging-loki.adoc` assembly. However, the path of that assembly is updated. So, I have updated the path on Line 3. Thanks!